### ISSUE TITLE
Feature Request: Biome whitelist

### ISSUE THREAD

**Issue by Duatao7055** [sam-kirby/drafty-kites#1]
A biome whitelist for this would be awesome for other themed stuff like heat vents or "windy" biomes.

**Comment by sam-kirby** [comment]
The mod already uses a biome whitelist; see drafty biomes in the config.

**Comment by sam-kirby** [comment]
Or am I misunderstanding - do you want the updrafts to be configurable such that they occur without requiring the player to be above the void?

**Comment by Duatao7055** [comment]
Yes, I meant that I want the updrafts to be configurable to work without the player being above the void. I just checked this for your responses now, which is why I didn't respond earlier. Also, I was being dumb and didn't think to check if there was a biome whitelist at all or a config, which is where some confusion came from, but also I just finally tested it out adding a biome without access to void to the whitelist and found out that it didn't work. So, yes, I would like it to be configurable that updrafts can occur without being directly above the void. Thank you for trying to understand what I meant, even though my question came from a lack of thinking and testing at first.
